### PR TITLE
Development

### PR DESCRIPTION
**Pull request by adriaanbd**

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+def bubble_sort(arr)
+	n = arr.length-1
+	(0..n).each do |i|
+		(0..n-i-1).each do |j|
```
Comment (heyapricot) [inline bubble_sort.rb]: [`..` defines an inclusive range and `...` an exclusive range. You can work with that](https://ruby-doc.org/core-2.6.1/Range.html)

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			if arr[j] > arr[j+1] then arr[j], arr[j+1] = arr[j+1], arr[j] end
```
Comment (heyapricot) [inline bubble_sort.rb]: Favor the use of the `if` modifier. You can write this one line statements as:
`arr[j], arr[j+1] = arr[j+1], arr[j] if arr[j] > arr[j+1]`

File: bubble_sort.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	length = arr.length - 1
+	swapped = false
+	(0..length).each do |i|
+		(0..length - i- 1).each do |j|
+			if arr[j] > arr[j + 1]
+				swapped = true
+				arr[j], arr[j + 1] = arr[j + 1], arr[j]
+			end
+		end
+		if swapped == false then break end
```
Comment (heyapricot) [inline bubble_sort.rb]: Favor the use of the if modifier

File: bubble_sort.rb (hunk 1 above)
Comment (mcrd25) [reply bubble_sort.rb]: Thanks, we missed notifications for this!

File: bubble_sort.rb (hunk 2 above)
Comment (mcrd25) [reply bubble_sort.rb]: Thanks, will do that

File: bubble_sort.rb (hunk 3 above)
Comment (mcrd25) [reply bubble_sort.rb]: Okay

**Pull request merged**